data_model/derive/src/model.rs

```diff
@@ -38,7 +29,7 @@ pub fn impl_model(emitter: &mut Emitter, input: &syn::ItemMod) -> TokenStream {
     quote! {
         #(#attrs)*
         #[allow(missing_docs)]
-        #vis_public #mod_token #ident {
+        #mod_token #ident {
```

Comment:
fair point, I was lazy. I will make them all private. I made this change because I generated the docs and saw this `model` module everywhere which wasn't appealing.

I can make `#model` macro also generate `pub use self::model::*;` but seeing that you don't like this implicit behavior I refrained from doing so. ok?

Reply:
Actually, generating `pub use self::model::*;` would be fine by me if the input to the macro wasn't a module, but just a set of items. No module in -> no visible module out

But I think this would be a very big change (other macros are calling it.. ) that would better go into a separate PR (_if_ we want to go ahead with it)

Reply:
I think it's fine like this